### PR TITLE
Enable auto formatting

### PR DESCRIPTION
**Pull request by kenoir**
For https://github.com/wellcometrust/platform/issues/4219

### CODE REVIEW

**Comment by alexwlchan** [comment]
You need to uncomment the "preflight" stage on L29 as well.

**Comment by alexwlchan** [comment]
Hmm, why is Travis still unhappy…

**Comment by alexwlchan** [comment]
Looking at the requests for Travis to build (https://travis-ci.org/wellcomecollection/stacks-service/requests, hamburger menu > requests), the error is:

> Could not parse wellcomecollection/stacks-service:.travis.yml@9ad83893b7bfc998

**Comment by alexwlchan** [comment]
And off we go…

**Review by alexwlchan — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 jobs:
   include:
-     - stage: preflight
-       env: TASK=travis-format
```
Comment (kenoir) [inline .travis.yml]: YA ML

**Pull request merged**